src/allowed-blocks-setup.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { map, without } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { getBlockTypes } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { JETPACK_DATA_PATH } from '../jetpack/projects/plugins/jetpack/extensions/shared/get-jetpack-data';
+import { registerBlock as registerJetpackLayoutGridBlock } from '../block-experiments/blocks/layout-grid/src';
```

Comment:
Okay. So explain something to me then, @fluiddot . Why is the name of the block type `jetpack/layout-grid` and not something like `experiment/layout-grid` or similar? I knew it wasn't part of the actual `jetpack` package, but the namespace of the block type didn't make practical sense to me.

Reply:
Good point, I noticed that the block references `jetpack` in its id so indeed looks it's part of Jetpack although it's not exposed from the Jetpack repository 🤔 , sorry for the misleading. Being that said, I agree with using the Jetpack reference in this block so I'll update the rest of my comments on the PR.

Reply:
No problem. It was just confusing why it would be related too `jetpack` but not be offered from the `jetpack` package. Is it something that should change ownership eventually from `block-experiments` to `jetpack`?

I'm unsure, but `block-experiments` feels like inherent technical debt if an "experiment" results in long-term production use instead of leveraging something like a feature toggle in the `jetpack` package for "beta" features.